content/en/docs/troubleshooting.md

```diff
@@ -1,6 +1,88 @@
+
+## When I verify an artifact, I get the error 'signature is not produced by a trusted signer'
+
+Assuming the trust store and trust policy are configured correctly, this error indicates that the signature is signed by an unknown identity, which should not be trusted. The verification should fail. On the other hand, Notation detects the problematic signature. Users should not use the artifact.
+
+Follow the steps to make sure the trust store and trust policy are configured correctly:
+
+1. Confirm the trust policy that you are using if you have multiple trust policies in `trustpolicy.json` file.
+2. Check the `trustStores` property, and make sure the value is correctly configured.
+3. Check the `trustedIdentities` property, and make sure the value is correctly configured. If the value is `"*"`, it means all the certificates stored in the trust stored (configured in `trustStores`) are trusted, then you need to make sure the certificates in the trust stores can be used to verify the signatures. If the value is in the format of x509 subject info, like `"x509.subject: CN=example.io,O=Notary,L=Seattle,ST=WA,C=US"`, then you need to make sure it is the identity that produces the signature. You can use command `notation cert show` to show the details of the certificate in the trust store. For example:
+
+```shell
+notation cert show --type ca --store mytruststore mycertificate.crt
+```
+
+An example of output:
+
+```text
+Certificate details
+--------------------------------------------------------------------------------
+Issuer: CN=example.io,O=Notary,L=Seattle,ST=WA,C=US
+Subject: CN=example.io,O=Notary,L=Seattle,ST=WA,C=US
+Valid from: 2023-01-15 07:55:01 +0000 UTC
+Valid to: 2023-01-16 07:55:01 +0000 UTC
+IsCA: false
+SHA1 Thumbprint: xxx
+```
+
+Check the `Subject` info in the output. If it is the identity that signs the artifact, you need to add the `Subject` info into `trustedIdentities`.
+
+## I have configured trust policy, but I still get the error 'no applicable trust policy'
+
+This error indicates that the `registryScopes` property is not correctly configured. This property contains a list of repository URIs, where the artifacts are stored. You need to make sure the signing artifact is stored in one of the listed repositories. If not, you need to add the missing repository URI in `registryScopes`, or you can add a new trust policy for the missing repository.
+
+The repository URI is in the format of `${registry-name}/${namespace}/${repository-name}`. For example, if the artifact to be verified is `registry.acme-rockets.io/software/net-monitor@sha256:xxx`, then the value for `registryScopes` should be `registry.acme-rockets.io/software/net-monitor`, the following values are wrong
+
+- `registry.acme-rockets.io`
+- `registry.acme-rockets.io/software`
```

Comment:
As a clarification, we should add that wildcards are not supported, and users need to list each repository explicitly. This is not a very user-friendly experience for customers who can have thousands of repositories :(